bokehjs/src/coffee/models/tools/gestures/wheel_zoom_tool.coffee

```diff
@@ -19,20 +19,20 @@ class WheelZoomToolView extends GestureTool.View
     @_scroll(e)
 
   _scroll: (e) ->
-    frame = @plot_model.get('frame')
+    frame = @plot_model.frame
     hr = frame.get('h_range')
     vr = frame.get('v_range')
```

Comment:
Yes any computed properties will have to use `get`. Fortunately there are not many of them and the nominal plan is to get rid of computed properties entirely.